resources/views/clubevent/createClubEventView.blade.php

```diff
@@ -1,54 +1,96 @@
 @extends('layouts.master')
 
 @section('title')
-	{{ trans('mainLang.createNewVEvent') }}
+        @if($createClubEvent)
+        	{{ trans('mainLang.createNewVEvent') }}
+        @else
+            {{ trans('mainLang.changeEventJob') }}
+        @endif
 @stop
 
 @section('content')
 
 @if(Session::has('userId'))
-	
-	{!! Form::open(['method' => 'POST', 'route' => ['event.store']]) !!}
+    <div class="row">
+        <div class="panel col-md-6 col-sm-12 col-xs-12 no-padding">
+            @if($createClubEvent)
+                {!! Form::open(['method' => 'POST', 'class'=>'form-inline','id'=>'templateSelectorForm']) !!}
+                <div class="form-group col-md-12 col-sm-12 col-xs-12 no-padding">
```

Comment:
This should be inside the main card and not on top of it and separated.

Reply:
this is to avoid browser messages that you have unchanged changes when choosing a template